.circleci/config.yml

```diff
@@ -653,40 +653,22 @@ workflows:
             - integration-test
           filters:
             branches:
-              only:
```

Comment:
It's probably worth talking about. I'm told we have leftover credits in circle for now, so I thought it might be useful to turn them on all the time. I was gonna take that out when I merged though, it was just to make testing easier/quicker as I was iterating.

Reply:
(Thinking about it, we don't usually need them that often unless it's for specific stuff. I'll revert that change before merging.)